### PR TITLE
Feat: Navigation component is added

### PR DESCRIPTION
**Pull request by kotalwar-krishna**
Navigation Related Css is you can find in nav-bar-component.css file.

Please review it

### CODE REVIEW

**Comment by rkapoor10** [comment]
> Navigation Related Css is you can find in nav-bar-component.css file.
> 
> Please review it

you may add screenshots as well

File: documentation/doc.css [hunk 1]
```diff
@@ -119,6 +119,11 @@ html {
     margin: 1rem 2rem;
 }
 
+/* nav-bar component */
+.margin-nav{
+    margin: 2rem;
+}
```
Comment (rkapoor10) [inline documentation/doc.css:125]: Why have you given nav-bar css in doc.css when you have nav-bar-component.css

Comment (rkapoor10) [inline documentation/doc.css:125]: Also, the margin-nav is general class and doesn't tell us about the exact margin. You may give utility classes for that give margins.

**Review by rkapoor10 — APPROVED** [review]
The code is good for Merge

File: documentation/doc.css (hunk 1 above)
Comment (kotalwar-krishna) [reply documentation/doc.css:125]: ok i will do

**Comment by kotalwar-krishna** [comment]
nav

File: documentation/doc.css (hunk 1 above)
Comment (kotalwar-krishna) [reply documentation/doc.css:125]: that nav bar for documentation page.

**Review by ShreyashMahajan — APPROVED** [review]
good to merge

**Pull request merged**